unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -103,11 +104,13 @@ public async UniTask<FriendRequest> RequestFriendship(string friendUserId, strin
 
         public void GetFriends(int limit, int skip) => apiBridge.GetFriends(limit, skip);
 
-        public void GetFriends(string usernameOrId, int limit) =>apiBridge.GetFriends(usernameOrId, limit);
+        public void GetFriends(string usernameOrId, int limit) => apiBridge.GetFriends(usernameOrId, limit);
 
-        public void GetFriendRequests(int sentLimit, int sentSkip, int receivedLimit,
-            int receivedSkip) =>
-            apiBridge.GetFriendRequests(sentLimit, sentSkip, receivedLimit, receivedSkip);
+        public async UniTask GetFriendRequests(int sentLimit, int sentSkip, int receivedLimit, int receivedSkip)
```

Comment:
it is missing the result in the task, for example `UniTask<List<FriendRequests>>`

Reply:
That's true, I was not returning any result because it was traveling through an event from `FriendsController`. Now, with the last refactor I have just added, this event is no longer needed and the result is directly managed by who make the call to `GetFriendRequests()`.

Please take a look again to my changes, thanks! 👍